esda/tests/test_moran.py

```diff
@@ -135,104 +135,412 @@ def test_Moran_Local_labels(self):
-       'Insignificant', 'Insignificant', 'Insignificant', 'Low-Low',
-       'Low-Low', 'Low-Low', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Low-Low', 'Insignificant', 'Low-Low',
-       'Insignificant', 'High-High', 'Insignificant', 'Low-Low',
-       'High-High', 'High-High', 'Insignificant', 'Low-High', 'Low-Low',
-       'Insignificant', 'Insignificant', 'Low-High', 'Low-Low', 'Low-Low',
-       'Insignificant', 'Low-Low', 'Insignificant', 'High-Low',
-       'Insignificant', 'Low-High', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Low-Low', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Low-Low', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'High-High', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Low-Low', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Low-Low', 'Insignificant',
-       'Insignificant', 'Insignificant', 'High-High', 'High-High',
-       'High-High', 'High-High', 'High-High', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Low-Low', 'Low-Low',
-       'Insignificant', 'Insignificant', 'High-High', 'Low-Low',
-       'Insignificant', 'Insignificant', 'High-High', 'Insignificant',
-       'Insignificant', 'Low-Low', 'High-High', 'Low-Low',
-       'Insignificant', 'Low-Low', 'Insignificant', 'Low-Low',
-       'Insignificant', 'Low-Low', 'Low-Low', 'Insignificant',
-       'Insignificant', 'Low-Low', 'Low-Low', 'Low-Low', 'Low-Low',
-       'Low-Low', 'Low-Low', 'Low-Low', 'Low-Low', 'Low-Low',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Low-Low',
-       'Low-Low', 'Insignificant', 'Insignificant', 'Low-Low', 'Low-Low',
-       'High-High', 'Insignificant', 'Low-Low', 'Insignificant',
-       'Low-Low', 'Low-Low', 'Low-Low', 'Low-Low', 'Low-Low', 'Low-Low',
-       'Low-Low', 'Low-Low', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Low-Low', 'Low-Low',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Low-Low', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Low-Low', 'Low-Low', 'Low-Low', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant',
-       'Insignificant', 'Insignificant', 'Insignificant', 'Insignificant']
+        expected_labels = np.array(
```

Comment:
Can we just check something like top 10 and then counts of each? This is looong.